demo/index.js

```diff
@@ -47,6 +47,16 @@ var plugin = function (config) {
 
     isRemovalOfLastAssocListItemAllowed(/*field*/) {
       return false;
+    },
+
+    createElement_Fieldset: function (props, ...children) {
```

Comment:
Yep, that's totally fine @neilff! I forgot, the `fieldset` helper doesn't correspond to a field that's passed in. Should have told you that you could just do a `{type: 'fields', fields: [...]}` wrapper around some fields. But this works too!